src/main/java/com/mentalhealthapi/mentalhealthapi/service/DisorderServiceStub.java

```diff
@@ -8,26 +8,26 @@
 
 @Component
 public class DisorderServiceStub implements IDisorderService {
-
-    /**
-     * Fetch all disorders
-     *
-     * @return
-     */
```

Comment:
Fix the JavaDoc instead of removing it.  Give a description of this method.  This is where the inline comments, below, belong.